src/Overview.tsx

```diff
@@ -69,16 +151,8 @@ export default function Overview() {
         </TableCell>
         <TableCell align="center">{obj.status}</TableCell>
         <TableCell align="center">{obj.location}</TableCell>
-        <TableCell align="center">
-          <Button
-            component={Link}
-            to={href}
-            size="small"
-            variant="outlined"
-            color="primary"
-          >
-            View
-          </Button>
+        <TableCell style={{ whiteSpace: 'pre-line' }} align="center">
```

Comment:
nit: assuming `pre-line` is to make `\n` work you could use basically
```
<Box display="flex" flexDirection="column">
  <div>Last seen:</div>
  <div>{{obj.last_updated}}</div>
</Box>
```
instead

Reply:
yup, that's reasonable